sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -133,6 +133,14 @@ def error(self, message):
     super().error(message)
 
 
+class _DictUnionAction(argparse.Action):
+  def __call__(self, parser, namespace, values, option_string=None):
+    if hasattr(namespace,
+               self.dest) and getattr(namespace, self.dest) is not None:
+      values.update(getattr(namespace, self.dest))
```

Comment:
when a flag is specified multiple times, usually the last occurrence takes precedence. I think we should initialize or update self.dest with new values , not the opposite. WDYT?

Reply:
Thanks. This makes sense. Will do and document.